container/types.proto

```diff
@@ -33,9 +33,16 @@ message Container {
   //
   // There are some "well-known" attributes affecting system behaviour:
   //
-  // * Subnet \
+  // * __NEOFS__SUBNET \
   //   String ID of container's storage subnet. Container can be attached to
   //   only one subnet.
+  //
+  // And some well-known attributes used by applications only:
+  //
+  // * Name \
+  //   Human-friendly name
+  // * LocalCreationUnixTime \
```

Comment:
Why local in `LocalCreationUnixTime`? I believe UTC timestamps are more appropriate for that.

Reply:
Regular UTC timestamp may be encoded in lots of different formats (ISO-8601, RFC 1123, etc) while UnixTime (still in UTC) in string presentation can be encoded consistently.

Reply:
Well, the question is whether the value is in local time or UTC. If this is in UTC, let's put the matter to rest.

Reply:
I agree that the name is misleading. It should reflect that this timestamp is created on user side without verification and may have any timestamp not related to reality.

Does anybody have suggestions on how to name it in a better way?

Reply:
I'd suggest not to bother with super-unombigous naming here as we simply cannot control it. A user can technically use our "well-known" attributes as they please, e.g. may write string `My birthday` to `LocalCreationUnixTime` and we must expect it to happen. Names of attributes are indicative and this one might be just `Timestamp`, for instance.